src/platform/with_wgl/native_gl_context.rs

```diff
@@ -0,0 +1,144 @@
+use platform::NativeGLContextMethods;
+use std::ffi::CString;
+use std::os::raw::{c_void};
+use std::ptr;
+use std::sync::{Once, ONCE_INIT};
+
+use winapi;
+use user32;
+use kernel32;
+use super::wgl;
+use super::wgl_attributes::*;
+use gleam::gl;
+
+/// Wrapper to satisfy `Sync`.
+struct HMODULEWrapper(winapi::HMODULE);
+unsafe impl Sync for HMODULEWrapper {}
+
+lazy_static! {
+    static ref GL_LIB: Option<HMODULEWrapper>  = {
+        let p = unsafe{kernel32::LoadLibraryA(b"opengl32.dll\0".as_ptr() as *const _)};
+        if p.is_null() {
+            debug!("opengl32.dll not found!");
+            None
+        }
+        else {
+            debug!("opengl32.dll LOADED!");
+            Some(HMODULEWrapper(p))
+        }
+    };
+}
+
+static LOAD_GL: Once = ONCE_INIT;
+pub fn load_gl() {
+    LOAD_GL.call_once(|| {
+        gl::load_with(|s| {
+            NativeGLContext::get_proc_address(s) as *const _
+        });
+    });
+}
+
+pub struct NativeGLContext {
+    pub render_ctx: winapi::HGLRC,
+    pub device_ctx:winapi::HDC, //
```

Comment:
nit: space after the colon. Also, either do comment, or remove the two slashes.

Please make the members private, since the intention is making `NativeGLContext` cross-platform. Feel free to add accessor methods if needed.